src/ruby/yast/wfm.rb

```diff
@@ -212,6 +212,24 @@ def self.call_builtin_wrapper(*args)
       !Mode.auto && Debugger.installed?
     end
 
+    # @param [CFA::AugeasParsingError] e the caught exception
+    # @return [String] human readable exception description
+    private_class_method def self.parsing_error_msg(e)
+      msg = "Parse error while reading file #{e.file}\n" \
+            "YaST cannot continue and will quit.\n" \
+            "\n" \
+            "Possible causes and remedies:\n" \
+            "1. You made a mistake when changing the file by hand,\n" \
+            "   the syntax is invalid. Try reverting the changes.\n" \
```

Comment:
as generic top level catch it is OK, bu still I do not like possible solution and outcome for users. I prefer way how it is done in bootloader, when this error happen it allows to repropose from scratch to get again working configuration - https://github.com/yast/yast-bootloader/blob/master/src/lib/bootloader/config_dialog.rb#L38

Reply:
OK. So you're saying that in addition to this, each user-facing module should have its own General Rescue, with proposing a working configuration from scratch as a possible option.

For the bootloader I see it as especially important since, well, can't boot, can't fix anything.
@mchf @teclator is there a similar option for the network module already?

Reply:
@mvidner well, it is not just about not booting cannot fix, but in general provide user option how to handle in user friendly way when configuration is broken for whatever reason. Or maybe we can suggest user to remove configuration file and force reinstall of package to get back default configuration and rerun yast module? not sure what is the best option from user POV

Reply:
Ah, network has already been improved in [yast/yast-network@0c717ce](https://github.com/yast/yast-network/commit/0c717ceb18e8f64d26b09f3e2136593700d8dc08#diff-a44342e8d97412ca7486bd2d61e8e429L269-R273), resulting in this:

![network_error_dialog](https://user-images.githubusercontent.com/102056/95775149-22aec580-0cc2-11eb-8b58-71eced0b8b84.png)